til-api/src/main/java/com/til/controller/interview/response/InterviewInfoResponse.java

```diff
@@ -0,0 +1,19 @@
+package com.til.controller.interview.response;
+
+import java.util.List;
+
+import com.til.domain.category.dto.CategoryDto;
+import com.til.domain.interview.dto.InterviewInfoDto;
+import com.til.domain.interview.dto.InterviewProblemQuestionDto;
+
+public record InterviewInfoResponse(
+                                    Long id,
+                                    List<CategoryDto> categoryList,
```

Comment:
P3: 카테고리 id 정보만 넘겨주면 좋을 것 같습니다!
```suggestion
                                    List<Long> categoryList 
```

Reply:
![12](https://github.com/user-attachments/assets/6100df95-5ba6-44a0-bc7c-96006ff8caae)
이 부분에 표시하려고 카테고리 통째로 받고있긴 합니다만, 프론트에서 store 활용하면 되는 문제인지 프론트쪽 이해도가 낮아서 잘 모르겠네요!

Reply:
프론트에 카테고리 store가 있어서 그거랑 매핑해서 출력될 수 있도록 하면 됩니다!

Reply:
현재 그렇게 사용되고 있는 페이지 예시 하나만 알려주실 수 있나요?

Reply:
기술학습 리스트, 상세 화면에서 활용하고 있는데 아래 PR 참고해보시면 좋을 것 같습니다! 프론트 작업하시다가 막히시면 호출해주세요~
https://github.com/SOMA-TIL/TIL-Frontend/pull/50